types/react/index.d.ts

```diff
@@ -2259,6 +2259,7 @@ declare namespace React {
         // Other HTML properties supported by SVG elements in browsers
         role?: string;
         tabIndex?: number;
+        crossOrigin?: "anonymous" | "use-credentials" | "";
```

Comment:
What's the purpose of the blank string as an allowed value here?

Reply:
The blank string appears in the `<img>` equivalent: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/833391a966801f2e2c4ca25466d81b5265e1f4f7/types/react/index.d.ts#L1957

and when used as an attribute value, [is treated the same as `anonymous`](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/crossorigin)

> Setting the attribute name to an empty value, like crossorigin or crossorigin="", is the same as anonymous.